contents/navigation/navbars/double-navbar/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  BarChart3Icon,
+  CalendarIcon,
+  FingerprintIcon,
+  GaugeIcon,
+  HomeIcon,
+  Settings,
```

Comment:
Settings should also be `<SettingsIcon />`

Reply:
Thanks.
I forgot. Fixed.